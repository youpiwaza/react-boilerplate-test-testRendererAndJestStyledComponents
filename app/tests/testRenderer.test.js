const React = require('react');
const ReactTestRenderer = require('react-test-renderer');

it('does things', () => {
  const inst = ReactTestRenderer.create(React.createElement('h1', null, 'qwe'));
  console.log(inst.toJSON());
  expect(inst.toJSON()).toMatchSnapshot();
});
