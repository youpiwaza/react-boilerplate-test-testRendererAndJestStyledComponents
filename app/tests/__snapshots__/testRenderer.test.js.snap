// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`does things 1`] = `
<h1>
  qwe
</h1>
`;
